helm/values-prod.yaml

```diff
@@ -173,6 +173,7 @@ microservice-chart:
     NODE_FORWARDER_READ_TIMEOUT: "10000"
     NODE_FORWARDER_CONNECTION_TIMEOUT: "10000"
     REDIRECT_PAYMENT_TYPE_CODES: "PPAYITR1XXX-RBPB,CHECKOUT-PPAYITR1XXX-RBPP,PPAYITR1XXX-RBPR"
+    DEPLOYMENT_VERSION: ""
```

Comment:
if the string is empy, you consider PROD as default in application.properties, correct? works?

Reply:
Good catch, no, an empty string would be used here, fixed with https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/322/commits/567a7bbc7bfd9205986d503dff973a6bbc12e6f6 to set it explicitly to PROD.